File: 1 - Iniciante/C99/BEE1038.c
#include <stdio.h>

int main(void){
	int cod, qtd;
	
	scanf("%d %d", &cod, &qtd);
	
	switch (cod){
		case 1:
			printf("Total: R$ %.2f\n", qtd * 4.00);
			break;
		case 2:
			printf("Total: R$ %.2f\n", qtd * 4.50);
			break;
		case 3:
			printf("Total: R$ %.2f\n", qtd * 5.00);
			break;
		case 4:
			printf("Total: R$ %.2f\n", qtd * 2.00);
			break;
		case 5:
			printf("Total: R$ %.2f\n", qtd * 1.50);
			break;
	}
	
	return 0;
}

File: 1 - Iniciante/Python 3.11/BEE1038.py
stream = input().split()
cod, qt = stream
cod, qt = int(cod), int(qt)

match cod:
    case 1:
        total = 4.00 * qt
    case 2:
        total = 4.50 * qt
    case 3:
        total = 5.00 * qt
    case 4:
        total = 2.00 * qt
    case 5:
        total = 1.50 * qt
    case _:
        exit()

print(f"Total: R$ {total:.2f}")